http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -934,11 +934,19 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig,
       clusterRequest.enableWelder.getOrElse(false))
     val replacements: Map[String, String] = clusterInit.toMap
 
+    val tmpCustomEnvVarFile = File.createTempFile("custom-env-vars", "env")
```

Comment:
It's probably possible to build this content as a `String` and upload to GCS directly, instead of creating a tmp file. It's already in memory in the `clusterRequest`.

Reply:
a la the templated resources at line ~970?